### ISSUE TITLE
Deal with withdrawn RORs

### ISSUE THREAD

**Issue by ahamelers** [datadryad/dryad-product-roadmap#3502]
From https://github.com/datadryad/dryad-product-roadmap/issues/3423

When ROR IDs are withdrawn we should remove them from being able to be selected in our dropdowns

@ryscher should we also replace instances of the withdrawn ROR IDs throughout our database with the 'successor' ids? We could potentially keep the old name strings...

**Comment by ryscher** [comment]
Yes, we should update our metadata for the withdrawn RORs. This involves both the DB update, and re-sending the metadata to DataCite.

**Comment by ahamelers** [comment]
ROR best practice is changing to the successor ROR IDs/names across the board (from community meeting chat).

<img width="455" alt="Screenshot 2024-07-16 at 17 47 45" src="https://github.com/user-attachments/assets/01261da8-775b-4466-a322-426ff2b57bb2">
